### PR TITLE
fix: Do not distribute metafiles

### PR DESCRIPTION
**Pull request by artem-shelkovnikov**
## Connected with https://github.com/aws/base64io-python/issues/38

Installing the `base64io` via `requirements.txt` in a Python project that we're running leads to README.rst, CHANGELOG.rst and LICENSE files from `base64io` replacing our files with the same names.

This PR aims to solve this problem, by not distributing these files with the installation - there seem to be no real need in this - LICENSE is distributed by default, while README and CHANGELOG are accessible via githun.com repository.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by tarekziade** [comment]
You do want to distribute then in the package, just not copy them in the current working dir at install time
You should just remove `data_files` and leave `MANIFEST.in` as it is

**Comment by tarekziade** [comment]
see the proposed change in https://github.com/aws/base64io-python/issues/38

**Review by texastony — CHANGES_REQUESTED** [review]
Thank you for the PR!
I think this is mostly correct.
~~But I need to dig into the Manifest documentation from Python.~~
I have read [PyPA's Manifest documentation](https://packaging.python.org/en/latest/guides/using-manifest-in/).

As @tarekziade has already pointed out, 
the Manifest should not change.

Only the data_files. 
If you update this PR by EOD of 2023/01/17 to not change the manifest,
only the data_file, 
I will approve it.

Otherwise, I will cut a new PR that only updates the data_files. 

This aligns with the other, more popular, Python libraries AWS Crypto Tools owns
which include the "meta" files in the manifest but not in the setup.py's data_file.

File: MANIFEST.in [hunk 1]
```diff
@@ -1,7 +1,3 @@
-include README.rst
-include CHANGELOG.rst
-include LICENSE
-
```
[the remark this answers is not in the document]
Comment (artem-shelkovnikov) [reply MANIFEST.in]: Thank you for the feedback! I've reverted the changes in MANIFEST.in, keeping only the change in `setup.py` file.

**Review by texastony — APPROVED** [review]

**Comment by texastony** [comment]
> @texastony what is the further process for the change? It looks like it can be merged when the checks complete, but they haven't yet started. Is there anything I can do?

@artem-shelkovnikov I am going to close and re-open this PR and that will run the checks.
(They had an issue, which has been resolved).

However, they will likely fail, due to changes in linters that are not related to this PR. 
I am working with my team to address this, or at least ignore the failing linter tests, which should allow your PR to be merged.

At this time, I do not believe any action is required on your part.

**Pull request merged**